src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingDocumentController.java

```diff
@@ -45,7 +49,7 @@
             @ApiResponse(code = 500, message = "Internal Server Error")})
     public ResponseEntity<AboutToStartOrSubmitCallbackResponse> generateHearingDocument(
             @RequestHeader(value = AUTHORIZATION_HEADER) String authorisationToken,
-            @NotNull @RequestBody @ApiParam("CaseData") CallbackRequest callback) {
+            @NotNull @RequestBody @ApiParam("CaseData") CallbackRequest callback) throws IOException {
```

Comment:
not sure we want to just throw exceptions here. at least catch the exception, log the error before returning an appropriate exception